CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+First of all, many thanks for considering making a contribution to this project.
+
+# Submitting PRs
+Please do not include any changes to `package-lock.json` in your PRs, even if you are updating package.json. These changes are almost impossible to review for security implications. If necessary, `package-lock.json` will be regenerated and committed by a maintainer after your PR is merged.
```

Comment:
```suggestion
Please do not include any changes to `package-lock.json` in your PRs, even if you are updating `package.json`. These changes are almost impossible to review for security implications. If necessary, `package-lock.json` will be regenerated and committed by a maintainer after your PR is merged.
```

Reply:
We can merge now, and I'll submit another PR with the changes